package.json

```
@@ -48,58 +48,45 @@
     }
   },
   "devDependencies": {
-    "@silverstripe/eslint-config": "0.0.5",
-    "@silverstripe/webpack-config": "^1.5.0",
+    "@babel/runtime": "^7.20.0",
+    "@silverstripe/eslint-config": "1.0.0-alpha6",
```

```suggestion
    "@silverstripe/eslint-config": "^1.0.0-alpha6",
```
We want to make sure we get updates